Example/VOKMockData/GET|-auth|Authorization=Token%20I_am_a_token&Content-Language=en.http

```diff
@@ -0,0 +1,3 @@
+HTTP/1.1 202 Accepted
```

Comment:
This was mostly just to match existing files - `200 OK` would probably be more appropriate, right?